pkg/inspector/image-inspector.go

```diff
@@ -99,46 +100,58 @@ func NewDefaultImageInspector(opts iicmd.ImageInspectorOptions) ImageInspector {
 
 // Inspect inspects and serves the image based on the ImageInspectorOptions.
 func (i *defaultImageInspector) Inspect() error {
-	client, err := docker.NewClient(i.opts.URI)
-	if err != nil {
-		return fmt.Errorf("Unable to connect to docker daemon: %v\n", err)
-	}
+	var scanReport []byte
+	var htmlScanReport []byte
+	var scanner iiapi.Scanner
 
-	if err = i.pullImage(client); err != nil {
-		return err
-	}
+	switch i.opts.ScanType {
+	case "openscap":
```

Comment:
yeah, another one on my TODO that I removed in this refactor just to lessen the changes.  We can make a separate PR to introduce that (and the switch logic) to reduce the junk in this.